segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -191,16 +152,13 @@ namespace pcl
         float dy = input_->points[idx1].y - input_->points[idx2].y;
         float dz = input_->points[idx1].z - input_->points[idx2].z;
         float dist = std::sqrt (dx*dx + dy*dy + dz*dz);
```

Comment:
I would actually just use eigen for it. I'll change it only if the rewrite is requested.